frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "antd": "^4.5.2",
```

Comment:
There is a warning that pops up when using the antd navbar that would be nice to fix. However, there is an [open](https://github.com/ant-design/ant-design/issues/22493) issue that they havent resolved on their end, so for now we cant do anything about it.

Reply:
Maybe we can disable strict mode for the navbar component like mentioned [here](https://github.com/facebook/react/issues/16362#issuecomment-520398080) . 

I will check it if it works.